### PR TITLE
[ign-transport1] Update BitBucket links

### PR DESCRIPTION
**Pull request by chapulina**
Signed-off-by: Louise Poubel <louise@openrobotics.org>

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ## Ignition Transport 1.x
 
 1. Fix issue #55.
-    * [Pull request 175](https://bitbucket.org/ignitionrobotics/ign-transport/pull-request/175)
+    * [BitBucket pull request 175](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-transport/pull-request/175)
```
Comment (maryaB-osr) [inline Changelog.md:4]: Trying out that link, it takes me to a general homepage for all the repos. This is the link that brings me straight to 175:
```suggestion
    * [BitBucket pull request 175](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-transport/pull-requests/175/page/1)
```

File: Changelog.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 1. Do not allow to advertise a topic that is currently advertised on the same node.
-   See [issue #54] (https://bitbucket.org/ignitionrobotics/ign-transport/issues/54)
-    * [Pull request 180](https://bitbucket.org/ignitionrobotics/ign-transport/pull-request/180)
+   See [issue #54] (https://github.com/ignitionrobotics/ign-transport/issues/54)
+    * [BitBucket pull request 180](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-transport/pull-request/180)
```
Comment (maryaB-osr) [inline Changelog.md:8]:
```suggestion
    * [BitBucket pull request 180](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-transport/pull-requests/180/page/1)
```

File: Changelog.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
 ### Ignition Transport 1.2.0
 
 1. Removed duplicate code in NetUtils, and improved speed of DNS lookup
-    * [Pull request 128](https://bitbucket.org/ignitionrobotics/ign-transport/pull-request/128)
+    * [BitBucket pull request 128](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-transport/pull-request/128)
```
Comment (maryaB-osr) [inline Changelog.md:13]:
```suggestion
    * [BitBucket pull request 128](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-transport/pull-requests/128/page/1)
```

File: doc/mainpage.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 This documentation provides useful information about the Ingition Transport library API.  Should you find problems with this documentation - typos, unclear phrases,
 or insufficient detail - please create a <a
-  href="https://bitbucket.org/osrf/ignition_transport/issues/new">new bitbucket issue</a>.
+  href="https://github.com/osrf/ignition_transport/issues/new">new GitHub issue</a>.
```
Comment (maryaB-osr) [inline doc/mainpage.html:5]:
```suggestion
  href="https://github.com/ignitionrobotics/ign-transport/issues/new">new GitHub issue</a>.
```

File: Changelog.md (hunk 1 above)
Comment (chapulina) [reply Changelog.md]: Nice catch! I wonder how many of these slipped in on the other repos :cold_sweat:

Comment (maryaB-osr) [reply Changelog.md]: I can leave a note about it in the channel so other people will double check

**Pull request merged**